src/modules/roc_packet/units.h

```diff
@@ -87,6 +87,9 @@ static inline size_t num_channels(channel_mask_t ch_mask) {
     return n_ch;
 }
 
+//! Packet block number in packet stream.
+typedef uint16_t blknum_t;
```

Comment:
It's worth to mention here that we're talking about FEC blocks.

Reply:
Done